torch/overrides.py

```diff
@@ -355,8 +355,8 @@ def get_testing_overrides() -> Dict[Callable, Callable]:
         torch.divide: lambda input, other, out=None: -1,
         torch.dot: lambda input, other, out=None: -1,
         torch.dropout: lambda input, p, train, inplace=False: -1,
-        torch.dsmm: lambda input, mat2: -1,
-        torch.hsmm: lambda mat1, mat2: -1,
+        torch.dsmm: lambda input, mat2: -1,  # type: ignore[attr-defined]
+        torch.hsmm: lambda mat1, mat2: -1,  # type: ignore[attr-defined]
```

Comment:
You added these to codegen, so ignores should be removed now?

Reply:
Can you remove as many as possible, and add some comments for any remaining ignores so it's clear what they're for?

Reply:
Done!